.travis.yml

```diff
@@ -51,3 +51,21 @@ matrix:
       script:
         - bash <(curl -s https://codecov.io/bash)
         - codecov
+
+    - language: generic
+      dist: trusty
+      sudo: false
+      script: skip
+      before_deploy:
+        - openssl aes-256-cbc -K $encrypted_0139fbcb2566_key -iv $encrypted_0139fbcb2566_iv
+          -in deploy_rsa.enc -out deploy_rsa -d
```

Comment:
Can you explain how these encrypted keys have been created? This should be documented in the file or somewhere else to help people recreate the setup.

Reply:
@norbusan Added a documentation file  🙂

Reply:
@norbusan Added a Documentation file 🙂

Reply:
Thanks